travis/install-common-deps.sh

```diff
@@ -33,12 +33,12 @@ pip install --upgrade pip==19.3.1
 
 # Install Python test dependencies only if we're running Python tests
 if [[ "$INSTALL_SMALL_PYTHON_DEPS" == "true" ]]; then
-  pip install --quiet -r ./travis/small-requirements.txt
-fi
-if [[ "$INSTALL_LARGE_PYTHON_DEPS" == "true" ]]; then
   # When downloading large packages from PyPI, the connection is sometimes aborted by the
   # remote host. See https://github.com/pypa/pip/issues/8510.
-  # As a workaround, we retry installation of large packages.
+  # As a workaround, we retry installation of packages.
+  retry-with-backoff pip install --quiet -r ./travis/small-requirements.txt
+fi
+if [[ "$INSTALL_LARGE_PYTHON_DEPS" == "true" ]]; then
   retry-with-backoff pip install --quiet -r ./travis/large-requirements.txt
   # Hack: make sure all spark-* scripts are executable. 
```

Comment:
This is how the code looks like:

https://github.com/mlflow/mlflow/blob/4a3133a7141adeaca593a99d0fc7924261576c33/travis/install-common-deps.sh#L35-L50